dart/lib/src/sentry_envelope.dart

```diff
@@ -0,0 +1,25 @@
+import 'sentry_envelope_header.dart';
+import 'sentry_envelope_item.dart';
+import 'protocol/sentry_event.dart';
+import 'protocol/sdk_version.dart';
+
+class SentryEnvelope {
+  SentryEnvelope(this.header, this.items);
+
+  final SentryEnvelopeHeader header;
+  final List<SentryEnvelopeItem> items;
+
+  static SentryEnvelope fromEvent(SentryEvent event, SdkVersion sdkVersion) {
+    return SentryEnvelope(SentryEnvelopeHeader(event.eventId, sdkVersion),
+        [SentryEnvelopeItem.fromEvent(event)]);
+  }
+
+  String serialize() {
```

Comment:
I agree, maybe should be called `toEnvelope()` too as we use `toJson` for Dict.
wondering if this should be returning a String or a Stream/Buffer whatsoever